src/components/user-settings-tabs/AccountSettings.js

```diff
@@ -53,9 +160,9 @@ class AccountSettings extends Component {
       session
     } = this.props
     getSession(session.authToken)
-    this.checkGithub()
-    this.checkHeroku()
-    this.checkTravis()
+    this.checkGithub().catch(console.error)
```

Comment:
Wrap these in a try catch block instead of doing .catch on each function

Reply:
That doesn't actually work because those functions are asynchronous. They return a Promise. Unless you use the async-await style (which doesn't make sense here), you can't use a try-catch block. You wouldn't want to use async-await because the three checks should be performed asynchronously, not as procedural.

Reply:
Can you do this?
```
try {
   await Promise.all[
      this.checkGithub(),
      this.checkHeroku(),
      this.checkTravis(),
   ]
} catch (e) {
   console.error(e)
}
```
That should be asynchronous. If that works, I think it's a lot cleaner than having a dangling catch at the end

Reply:
Cleaner? Really? You're adding 6 new lines! I'll do it just to get you off my back, but I don't agree with it! :-P